internal/backends/winit/glwindow.rs

```diff
@@ -456,8 +454,10 @@ impl<Renderer: WinitCompatibleRenderer + 'static> WindowAdapterSealed for GLWind
                 if let Some(requested_size) = requested_size {
                     // It would be nice to bound this with our constraints, but those are in logical coordinates
                     // and we don't know the scale factor yet...
+                    let physical_size =
+                        requested_size.to_physical(scale_factor_override.unwrap_or(1.0) as f32);
                     window_builder.with_inner_size(winit::dpi::Size::new(
```

Comment:
here, inner_size takes a `winit::dpi::Size` which is also an enum logical/physical.

you should not convert the `requested_size` to a physical size,  you should just convert it to the `dpi::Size` with the same logical or physical thing.

Reply:
I was thinking about that, but decided against it: I wanted to do all scale factor conversions using the same code paths. I was afraid to end up with rounding errors or something if they do not produce absolutely identical results.

Reply:
But that's the whole point of this change: let winit decide the scale factor when it puts the window on screen so that the window has the right size from the start. 
This is the window_builder, the scale factor is not known yet.

In fact there are two cases:
 1. When scale_factor_override is set, this code is correct,  then we need to give the physical size always
 2. when the override is not set (common case), we must pass the logical or phisical dirrectly.

Reply:
I think the newest version does the right thing now:-)